workspaces/marketplace/plugins/marketplace/src/components/Markdown.tsx

```diff
@@ -35,6 +44,49 @@ export const Markdown = (props: MarkdownProps) => {
     return href;
   };
 
+  useEffect(() => {
+    document.querySelectorAll('pre code').forEach(codeBlock => {
+      const pre = codeBlock.parentElement;
+
+      if (!pre) return;
+      if (pre.querySelector('.copy-button')) return;
+
+      const button = document.createElement('button');
+      button.className = 'copy-button';
+      Object.assign(button.style, {
+        position: 'absolute',
+        top: '8px',
+        right: '8px',
+        padding: '2px',
+        color: '#6a6e73',
```

Comment:
can we use an MUI token here instead? e.g., `palette.text.secondary`

Reply:
This is a button tag not the Mui Button component, so the `text.secondary` doesn't work unfortunately.

Reply:
does `useTheme` work?